server/lib/ocr/ExpenseOCRService.ts

```diff
@@ -1,20 +1,25 @@
-export interface OCRParseResult {
+export interface ExpenseOCRParseResult {
   confidence: number;
   description: string;
   amount: { value: number; currency: string };
   date: Date;
+  raw: Record<string, any>;
+  items: Array<{
+    description: string;
+    amount: { value: number; currency: string };
+    url: string;
+    incurredAt: Date;
+  }>;
 }
 
 /**
  * An OCR service interface for expenses attachments (invoices, receipts, etc.)
  */
 export interface ExpenseOCRService {
-  /**
-   * Processes an image and returns the extracted content.
-   */
-  processImage: (images: Buffer | Buffer[]) => Promise<OCRParseResult[]>;
```

Comment:
Good, I was doubtful we needed both methods when I first reviewed this mock.

Reply:
Yep, it could be a way to speed things up by eventually uploading both to S3 and Klippa simultaneously, but it's overkill at this point.